src/javaRosa.js

```diff
@@ -1388,6 +1388,9 @@ define([
                     insertOutputRef(_this, target, path, mug);
                     if (window.analytics) {
                         window.analytics.usage("Output Value", "Drag and Drop");
+                        if (path.contains('casedb')) {
+                            window.analytics.usage("Case Management", "Form Builder", "Click and Drag");
```

Comment:
Good catch @millerdev, I forgot that I already had analytics for a similar event. Let's make the names consistent (so, "Drag and Drop")